.gitignore

```diff
@@ -24,6 +24,8 @@ junit.xml
 npm-error.log
 yarn-error.log
 
+dbdir/
+
```

Comment:
this was necessary for websocket development. IT should be good to remove once we standup the API

Reply:
I ended up removing it since this shouldn't be relevant for the current configuration as we're not persisting data with the current configuration